### ISSUE TITLE
Scrolling issues with pinning enabled

### ISSUE THREAD

**Issue by iorzetic** [angular-ui/ng-grid#3317]
There seems to be a weird problem with scrolling in the grid when pinning the columns is enabled. We experience this problem in our application but this is also very easy to reproduce on the examples provided on ui-grid.info.
Here are some reproduction steps:
1. Open the example page for pinning: http://ui-grid.info/docs/#/tutorial/203_pinning
2. Go to the bottom of the page where the grid is. The grid here as two pinned columns, one to the left and one to the right.
3. Positon the mouse pointer over the first pinned column (left) or over the main content of the grid which is not pinned and try to use the mousewheel to scroll -> notice that the scroll does not work.
4. Position the mouse pointer over the last column (pinned right) and use the mousewheel to scroll -> notice that the scrolling now works
5. Notice that after you scrolled successfully at step 4, you can now scroll with no difficulty when having the pointer over the other columns in the grid as well (which didn't work before)
6. Even more weird than this: if you scroll again to the top of the list, the scrolling breaks again and we have the same situation as the one in the beginning (scrolling only works when the mouse pointer is over the last pinned column at first, but once scrolled a bit, it starts working when having the pointer over the other columns).

I have to mention that in our application we only pin columns to the left, so the right part of the grid is all filled with unpinned columns, and this problem is still reproducible: the scroll works only if you initially scroll over the unpinned columns area.
Basically, the pattern that I identified would be that the scrolling works initially only on the right most "container" (where "container" means an area with either pinned or unpinned columns).

Let me know if you need me to provide another plunkr or other reproduction steps that would help you.

Also I have to mention that I am using Chrome with the latest version available (41), but I was able to reproduce this with IE (11) and Mozilla Firefox (37).

**Comment by PaulL1** [comment]
Duplicate of #3260.

**Comment by iorzetic** [comment]
Sorry I didn't identify this correctly with the other open issue. The "trackpad" in the name of the other one threw me off.

Maybe it is useful as information for debugging and solving this issue that the scrolling doesn't work only when we are at the top (I didn't saw this mentioned in the other task description)

**Comment by swalters** [comment]
I can't recreate this issue on latest master.  @PaulL1 can you?

**Comment by PaulL1** [comment]
I can't exactly recreate that, but on the 203 tutorial I get a little different behaviour between the body and pinned left.  It's nowhere near as noticeable as it was, but on the pinned container if I scroll aggressively it sometimes propagates to the parent, whereas on the body container no matter how hard I scroll (with trackpad) it doesn't seem to propagate to parent.  I'm not sure if that's enough to be worth worrying about though - certainly it could go to 3.1.

**Comment by iorzetic** [comment]
I can still reproduce this on the tutorial page (which I suppose it is using the latest unstable release). The scroll works correctly only when one uses the right-most "container":
http://screencast.com/t/jysCb3Q1Lx

Let me know if I can provide more details on this.

**Comment by swalters** [comment]
will fix in #3772

**Comment by phicho** [comment]
This is fixed but unfortunately for tablets now you can "scroll" (list the table content only on the last pinned column) on the rest of the columns its not working. 
Its easy to reproduce just go to  http://ui-grid.info/docs/#/tutorial/203_pinning and in chrome open the developer console and switch the device mode to nexus 7.
If someone knows quick fix or workaround please do tell.

**Comment by piyush-dholariya** [comment]
Still it does not allow me to scroll till bottom of grid,
2nd issue is when I do scrolling upwards/downwards other row seems blinking or looks like removed for while, its not giving smooth scroll.
